src/scss/layout.scss

```diff
@@ -0,0 +1,64 @@
+@import 'variables/sizes';
+
+.main-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.main-body {
+  max-width: 800px;
+  width: 100%;
+}
+
+.main-body-medium {
+  max-width: 600px;
+  width: 100%;
+}
+
+.pad-l-three {
```

Comment:
Try something like what is proposed [here](https://github.com/Wolox/layout/pull/1)